tests/ui_tests/test_configs.py

```diff
@@ -359,39 +383,66 @@ def test_configs_fields(sdk_client_fs: ADCMClient, config_dict, app_fs):
     field_type = config['config'][0]['type']
 
-    cluster, ui_config = prepare_cluster_and_get_config(sdk_client_fs, path, app_fs)
-
-    fields = ui_config.get_app_fields()
+    cluster, config_page = prepare_cluster_and_config(sdk_client_fs, path, app_fs)
     with allure.step('Check save button status'):
-        save_err_mess = f"Correct status for save button {[expected['save']]}"
-        button_state = ui_config.save_button_status()
-        assert expected['save'] == button_state, save_err_mess
-        if expected['save']:
-            ui_config.save_configuration()
-            ui_config.assert_no_popups_displayed()
+        expected_state = expected['save']
+        assert (
+            not (config_page.config.is_save_btn_disabled()) == expected_state
+        ), f'Save button should{" not " if expected_state is False else " "}be disabled'
+        if expected_state:
+            config_page.config.save_config()
+
     with allure.step('Check field configuration'):
         if expected['visible']:
             if expected['visible_advanced']:
-                assert not fields, "Config fields presented, expected no"
-                ui_config.show_advanced()
-                assert ui_config.advanced, 'Advanced button not clicked'
-            fields = ui_config.get_app_fields()
+                config_page.config.check_no_rows_or_groups_on_page()
+                config_page.config.click_on_advanced()
+            fields = config_page.config.get_all_config_rows()
             assert fields, 'No config fields, expected yes'
             for field in fields:
-                ui_config.assert_field_is_editable(field, expected['editable'])
+                expected_editable_state = expected['editable']
+                assert (
+                    config_page.config.is_element_editable(field) == expected_editable_state
+                ), f"Element should{' not ' if expected_editable_state is False else ' '}be editable"
             if expected['content']:
-                ui_config.assert_field_content_equal(cluster, field_type, fields[0], config['config'][0]['default'])
+                if field_type == 'boolean':
+                    config_page.config.assert_bool_value_is(field, expected_value=config['config'][0]['default'])
+                elif field_type in ("password", "secrettext"):
+                    is_password_value = True if field_type == "password" else False
+                    config_page.config.assert_input_value_is(
+                        expected_value='********', display_name=field_type, is_password=is_password_value, is_list=False
+                    )
+                elif field_type == "list":
+                    config_page.config.assert_input_value_is(
+                        expected_value=config['config'][0]['default'],
+                        display_name=field_type,
+                        is_password=False,
```

Comment:
as I recall `is_password` is False by default

Reply:
removed